skyuxconfig.json

```diff
@@ -34,7 +34,8 @@
   },
   "app": {
     "styles": [
-      "@skyux/docs-tools/css/docs-tools.css"
+      "@skyux/docs-tools/css/docs-tools.css",
+      "src/app/visual-styles.scss"
```

Comment:
This should only be added to the `skyuxconfig-e2e.json` file.

Reply:
The intent here was to let the visual demos be a bit more legible for devs doing manual testing. Wouldn't we want that in the `skyuxconfig.json` as well? Or should we just be putting those styles in the visual component itself?

Reply:
This stylesheet will "leak" into the docs pages as well. It might create confusion for future engineers who think the `visual-styles.scss` file only affects the visual tests. Maybe create a component stylesheet for the visual tests instead? Or, create a new global stylesheet that's named in such a way that it can be used for the "production" pages, too.

Reply:
Good points. I moved it to a component-specific style sheet.